submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,174 @@
+
+As for the other fluids, you want to ensure that the water from your offshore pumps is reaching your heat exhangers without significant [throughput loss in pipelines](https://wiki.factorio.com/Fluid_system#Pipelines), hence make sure to have multiple pipelines along the way. While it is ideal to simply have short pipelines, you can also consider using pumps. The critical number is a maximum of 17 pipe segments between every 2 pumps, in order to maintain the 1200 water/second from an offshore pump. Here it is worth noting that underground pipes pairs at any distance count as only 2 pipe segments. Secondly, you want to make sure the steam generated has enough pipe space to flow from exchangers to turbines without backing up, which is not an issue except for enormous nuclear plants and can always be solved again using more pipes or adding pumps. 
+
+### What about storing the steam?
+
+There is some debate in the community about whether steam storage is worthwhile. It is worth noting that it is not necessary for normal plant operation, since a correct ratio of exchangers to turbines (with *just* more than enough turbines) would ensure any steam produced is being consumed while the plant is running at full capacity. Meanwhile, adding optional components for steam storage, especially at a large scale, impacts the UPS by adding more fluid calculations. The size of this impact is arguably negligibile compared to that of heat exchangers and steam turbines, based on my examinations and [these](https://www.reddit.com/r/technicalfactorio/comments/qc0npz/the_impact_of_nuclear_power_plants_on_ups/) [two](https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/) detailed explorations by technical community members (further discussed below). Meanwhile, steam storage offers some optional benefits, but whether you prefer these benefits over a little extra UPS is up to you:

```

Comment:
maybe you can name "steam batteries", as readers might have already heard of this elsewhere?

Reply:
So there is a previous name for it? Sure !